lib/ueberauth/strategy/github.ex

```diff
@@ -158,7 +158,7 @@ defmodule Ueberauth.Strategy.Github do
     %Info{
       name: user["name"],
       nickname: user["login"],
-      email: user["email"],
+      email: user["email"] || Enum.find(user["emails"] || [], &(&1["primary"]))["email"],
```

Comment:
Is there a case where the return of `Enum.find(user["emails"] || [], &(&1["primary"]))` could be nil? Wondering if it would be prudent to use a `Map.get` on this value